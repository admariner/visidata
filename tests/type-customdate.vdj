#!vd -p
{"sheet": "global", "row": "disp_formatter", "longname": "set-option", "input": "python", "keystrokes": ""}
{"longname": "open-file", "input": "sample_data/sample.tsv", "keystrokes": "o"}
{"sheet": "sample", "col": "", "row": "", "longname": "select-rows", "input": "", "keystrokes": "gs", "comment": "select all rows"}
{"sheet": "sample", "col": "OrderDate", "row": "0", "longname": "setcol-input", "input": "05/1991", "keystrokes": "ge", "comment": "set contents of current column for selected rows to same input"}
{"sheet": "sample", "col": "", "row": "", "longname": "unselect-rows", "input": "", "keystrokes": "gu", "comment": "unselect all rows"}
{"sheet": "sample", "col": "OrderDate", "row": "", "longname": "type-customdate", "input": "%m/%Y", "comment": "set type of current column to custom date format"}
