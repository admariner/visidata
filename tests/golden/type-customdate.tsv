OrderDate	Region	Rep	Item	Units	Unit_Cost	Total
1991-05-01	East	Jones	Pencil	95	1.99	189.05
1991-05-01	Central	Kivell	Binder	50	19.99	999.50
1991-05-01	Central	Jardine	Pencil	36	4.99	179.64
1991-05-01	Central	Gill	Pen	27	19.99	539.73
1991-05-01	West	Sorvino	Pencil	56	2.99	167.44
1991-05-01	East	Jones	Binder	60	4.99	299.40
1991-05-01	Central	Andrews	Pencil	75	1.99	149.25
1991-05-01	Central	Jardine	Pencil	90	4.99	449.10
1991-05-01	West	Thompson	Pencil	32	1.99	63.68
1991-05-01	East	Jones	Binder	60	8.99	539.40
1991-05-01	Central	Morgan	Pencil	90	4.99	449.10
1991-05-01	East	Howard	Binder	29	1.99	57.71
1991-05-01	East	Parent	Binder	81	19.99	1619.19
1991-05-01	East	Jones	Pencil	35	4.99	174.65
1991-05-01	Central	Smith	Desk	2	125.00	250.00
1991-05-01	East	Jones	Pen Set	16	15.99	255.84
1991-05-01	Central	Morgan	Binder	28	8.99	251.72
1991-05-01	East	Jones	Pen	64	8.99	575.36
1991-05-01	East	Parent	Pen	15	19.99	299.85
1991-05-01	Central	Kivell	Pen Set	96	4.99	479.04
1991-05-01	Central	Smith	Pencil	67	1.29	86.43
1991-05-01	East	Parent	Pen Set	74	15.99	1183.26
1991-05-01	Central	Gill	Binder	46	8.99	413.54
1991-05-01	Central	Smith	Binder	87	15.00	1305.00
1991-05-01	East	Jones	Binder	4	4.99	19.96
1991-05-01	West	Sorvino	Binder	7	19.99	139.93
1991-05-01	Central	Jardine	Pen Set	50	4.99	249.50
1991-05-01	Central	Andrews	Pencil	66	1.99	131.34
1991-05-01	East	Howard	Pen	96	4.99	479.04
1991-05-01	Central	Gill	Pencil	53	1.29	68.37
1991-05-01	Central	Gill	Binder	80	8.99	719.20
1991-05-01	Central	Kivell	Desk	5	125.00	625.00
1991-05-01	East	Jones	Pen Set	62	4.99	309.38
1991-05-01	Central	Morgan	Pen Set	55	12.49	686.95
1991-05-01	Central	Kivell	Pen Set	42	23.95	1005.90
1991-05-01	West	Sorvino	Desk	3	275.00	825.00
1991-05-01	Central	Gill	Pencil	7	1.29	9.03
1991-05-01	West	Sorvino	Pen	76	1.99	151.24
1991-05-01	West	Thompson	Binder	57	19.99	1139.43
1991-05-01	Central	Andrews	Pencil	14	1.29	18.06
1991-05-01	Central	Jardine	Binder	11	4.99	54.89
1991-05-01	Central	Jardine	Binder	94	19.99	1879.06
1991-05-01	Central	Andrews	Binder	28	4.99	139.72
